pipeline/lovd/combineEquivalentVariantSubmissions.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+"""
+Zack Fischmann
```

Comment:
Not sure what the standard should be, but in general there is github to take care of authorship etc.